README.md

```diff
@@ -4,7 +4,8 @@
 
 ![OpenELB Logo](./doc/logo/color-horizontal.svg)
```

Comment:
I absolute want to update this logo, but i haven't the new logo.

Reply:
Let me replace it.